### ISSUE TITLE
URL links http not https

### ISSUE THREAD

**Issue by JMurphyWeb** [karmaradio/karma#612]
Some users are not able to follow links from emails as they are being created as http.

**Comment by JMurphyWeb** [comment]
I have swapped out all instances of `Engine.Router.Helpers` `_url` and swapped it with the helper function in our `Controllers.Helpers` to get the base url from the env variable which has `https` hardcoded.

Not an ideal solution, our config should generate https urls but we needed a quick fix.

**Comment by JMurphyWeb** [comment]
@jonscottfilm can we close this?

**Comment by jonscottfilm** [comment]
Yes, unless we feel we should clean up the errant double backslash after .com? (in case of future repercussions?)

https://karmaradio-staging.herokuapp.com//projects/36/offers/114
